src/graphql/resolvers/global-descriptor-resolvers.ts

```diff
@@ -0,0 +1,160 @@
+import { IResolvers } from 'graphql-tools';
+import services from '../../services';
+import { IContext } from '../context';
+import { PubSub } from 'graphql-subscriptions';
+import { GlobalDescriptorService } from '../../services/global-descriptor-service';
+
+export interface IDescriptor {

```

Comment:
A lot of things like this are reused, can we import them instead of redoing all this?

Reply:
? IDescriptor

Reply:
Yes, I believe I use this interface in my file, so you can reuse that instead of re writing.